### ISSUE TITLE
[Bug] Unable to save rating rule

### ISSUE THREAD

**Issue by soleilvermeil** [eikofee/eikonomiya-front#23]
**Describe the bug**
Pressing the floppy disk icon when trying to save artifacts' rating rules throws an error:
```
[Error: ENOENT: no such file or directory, scandir '/app/data/api/characters'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'scandir',
  path: '/app/data/api/characters'
}
[Error: An error occurred in the Server Components render. The specific message is omitted in production builds to avoid leaking sensitive details. A digest property is included on this error instance which may provide additional details about the nature of the error.] {
  digest: '419553095'
}
```

**To Reproduce**
1. Press the floppy disk icon.

**Additional context:**
I freshly rebuilt the docker image and the folder to which I mapped `/app/data` does not contain any `api/` folder. Instead:
```
<mapped folder>/
  <uid1>/
    characters/
    rules/
    player
  <uid2>/
  ...
```

**Comment by eikofee** [comment]
yeah I removed the api/ folder for now because of some issues during build step of nextjs.
working on it

**Comment by eikofee** [comment]
should be fixed in 3f82a7381a9.

**Comment by soleilvermeil** [comment]
I confirm it is working now!
